fix: Conflict in generation of MDC http.request.url

jeanpommier opened the pull request:
Fixes https://github.com/georchestra/georchestra-gateway/issues/203

pmauduit reviewed (approved):
LGTM overall, but reading #203, I am surprised not to find any `MDC.put()` line removed, as if I understand correctly, it was called twice on adding the url ?

jeanpommier commented:
No put removed because I renamed it.
As stated in the issue, the second one actually provides the full URL, but the first one did provide only the path (not scheme, no host, no request params, just the raw path). Which is why I renamed it `path`. And removed nothing.

jeanpommier commented:
Any objection to merging it @groldan  ?
